examples/spells/basicSpells.qnt

```diff
@@ -75,6 +75,16 @@ module basicSpells {
     assert(Set() == Set().setRemove(3)),
   }
 
+  pure def setAdd(__set: Set[a], __elem: a): Set[a] = {
```

Comment:
Can you add a doc comment?

Reply:
Could this be instead added as a built-in named `add`?

Reply:
Yes, good point! It's actually in the plans to change Quint's type system to allow overloading of names for different types, so we should be able to get rid of all `setRemove`, `mapRemove`, `listRemove` etc and just have `remove` as everyone expects.

Reply:
> It's actually in the plans to change Quint's type system to allow overloading of names for different types

That would be amazing!